src/components/home/Events.jsx

```diff
@@ -25,48 +24,40 @@ const Events = () => {
           </Typography>
           <Paper {...leftSidePaperStyles}>
             <Typography p={2}>
-              CODE PDX meets up in-person once a month for a project demo night and onboarding
-              event. We also occasionally do social meetups.
-              <br />
-              <br />
-              Not sure if we need three boxes. I Need to find more content to put here or remove a
-              bubble and maybe toss in a blob since this will be mid page.
+              CODE PDX meets up in-person twice a month at GRAYBOX in Portland. Once for a project
```

Comment:
Could maybe add a link to the GRAYBOX website.

Reply:
added